tests/unit/template_cassdc_test.go

```diff
@@ -280,12 +286,40 @@ var _ = Describe("Verify CassandraDatacenter template", func() {
 			// Second container should be medusa
 			Expect(cassdc.Spec.PodTemplateSpec.Spec.Containers[1].Name).To(Equal(MedusaContainer))
 
-			// Verify volumeMounts and volumes
-			Expect(len(cassdc.Spec.PodTemplateSpec.Spec.Containers[1].VolumeMounts)).To(Equal(4))
-			Expect(cassdc.Spec.PodTemplateSpec.Spec.Containers[1].VolumeMounts[0].Name).To(Equal(medusaConfigVolumeName))
+			medusaContainer := GetContainer(cassdc, MedusaContainer)
+			medusaConfigMap := HelmReleaseName + "-medusa"
 
-			Expect(len(cassdc.Spec.PodTemplateSpec.Spec.Volumes)).To(Equal(3))
-			Expect(cassdc.Spec.PodTemplateSpec.Spec.Volumes[0].Name).To(Equal(medusaConfigVolumeName))
+			Expect(kubeapi.GetVolumeMountNames(medusaContainer)).To(ConsistOf(medusaConfigMap, "cassandra-config", "server-data", storageSecret))
```

Comment:
You probably saw that from unit tests failing, but there's no GetVolumeNames, so maybe you forgot to commit that file?

Reply:
yes 🤦 and committed now.